packages/server-wallet/package.json

```diff
@@ -81,6 +81,8 @@
     "db:rollback": "npx knex migrate:rollback --cwd src/db-admin",
     "db:seed": "npx knex seed:run --cwd src/db-admin",
     "db:setup:dev": "yarn db:drop && yarn db:create && yarn db:migrate",
+    "db:reset": "yarn db:drop; yarn db:create && yarn db:migrate",
+    "db:reset:test": "(NODE_ENV=test yarn db:reset)",
```

Comment:
Is there any reason why we do different things based on `NODE_ENV` being `test` or `development`?

Reply:
It's just a convenience script to set the shell variable

Reply:
Yeah but I'm just wondering why we need to even care about this at all — is there a fundamental reason why we need two environments

Reply:
There are different databases. The database used depends on the node env

Reply:
Yeah I know but I am wondering why we need to use different databases based on the node env